git/git_test.go

```diff
@@ -39,50 +39,43 @@ func Test_UncommittedChangeCount(t *testing.T) {
 }
 
 func Test_CurrentBranch(t *testing.T) {
-	cs, teardown := test.InitCmdStubber()
-	defer teardown()
-
-	expected := "branch-name"
-
-	cs.Stub(expected)
-
-	result, err := CurrentBranch()
-	if err != nil {
-		t.Errorf("got unexpected error: %w", err)
-	}
-	if len(cs.Calls) != 1 {
-		t.Errorf("expected 1 git call, saw %d", len(cs.Calls))
+	type c struct {
+		Stub     string
+		Expected string
 	}
-	if result != expected {
-		t.Errorf("unexpected branch name: %s instead of %s", result, expected)
+	cases := []c{
+		{
+			Stub:     "branch-name",
+			Expected: "branch-name",
+		},
+		{
+			Stub:     "refs/heads/branch-name",
```

Comment:
This is great! Could you also add a stub that has non-breaking spaces in a name? I believe this would be `\u00A0` in Go syntax. This is to verify that us splitting by newlines would keep these spaces intact 👌